containers/runtime/xds/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.16
+
+RUN mkdir -p /src/workspace
+WORKDIR /src/workspace
+
+# when BREAK_CACHE arg is set to a random value (e.g. by "--build-arg BREAK_CACHE=$(uuidgen)"),
+# it makes sure the docker cache breaks at this command, and all the following
+# commands in this Dockerfile will be forced to re-run on each build.
+# This is important to ensure we always clone the repository even if "GITREF" stays unchanged
+# (important e.g. when GITREF=master, when the clone command could get cached and
+# we'd end up with a stale repository).
+ARG BREAK_CACHE
```

Comment:
Would the [`--no-cache`](https://docs.docker.com/engine/reference/commandline/build/#:~:text=instructions%20during%20build-,%2D%2Dno%2Dcache,-Do%20not%20use) arg to `docker build` work here?

Reply:
I think `--no-cache` should work great here, since we really don't have much things to keep before installing copy step, always starting from scratch makes scene. Thank you for letting me know!